lib/twilio/util.ts

```diff
@@ -62,6 +62,12 @@ function isFirefox(navigator?) {
     && /firefox|fxios/i.test(navigator.userAgent);
 }
 
+/**
+ * @privateRemarks
```

Comment:
Do we currently use this in our codebase? If not, can we remove it to be consistent? I understand it's probably better but it's worse if we're not being consistent.